### DISCUSSION TITLE
Osu!Lazer not working

### DISCUSSION THREAD

**Discussion by SaiTheDuck** [ppy/osu#17845] [Q&A]
I downloaded osu lazer and tried to open the app but nothing happened, been looking around for solutions but found none. if you please help that would be great!

**Comment by integerrr** [comment]
Please have a read through #15188 and #16869 first. We cannot offer any help if you don't provide any information

**Reply by SaiTheDuck** [reply]
[runtime.log](https://github.com/ppy/osu/files/8500552/runtime.log)
